src/Store.spec.lua

```diff
@@ -168,7 +168,7 @@ return function()
 
 			expect(caughtState.Value).to.equal(1)
 			expect(caughtAction.type).to.equal("@@INIT")
-			expect(caughtErrorResult.message).to.equal("Caught error in reducer with init")
+			expect(string.find(caughtErrorResult.message, "Caught error in reducer ")).to.be.ok()
```

Comment:
You're not including the reducer name to avoid flakiness, right?  I understand the switch to `.to.be.ok()`, but you're not checking for the specific reducer.

Reply:
correct. I didn't want to get into forking the test behavior depending on Luau vs Lua 5.1, but I could if it's helpful.